src/main/java/io/fabric8/elasticsearch/plugin/OpenShiftElasticSearchService.java

```diff
@@ -65,31 +66,53 @@ protected OpenShiftElasticSearchService(final Settings settings, final Client es
 		this.cache = cache;
 		this.aclNotifier = aclNotifer;
 		restController.registerFilter(aclFilter);
-		restController.registerFilter(new KibanaUserReindexFilter(settings, logger));
+		
+		boolean reindexEnabled = settings.getAsBoolean(OPENSHIFT_KIBANA_REWRITE_ENABLED_FLAG, OPENSHIFT_KIBANA_REWRITE_ENABLED_DEFAULT);
+		
+		
+		if ( reindexEnabled ) {
+			restController.registerFilter(new KibanaUserReindexFilter(settings, logger));
+			logger.debug("Starting with Kibana reindexing feature enabled");
+		}
+		else {
+			logger.debug("Starting with Kibana reindexing feature disabled");
+		}
 	}
 
 	@Override
 	protected void doStart() throws ElasticsearchException {
+		
+		boolean dynamicEnabled = settings.getAsBoolean(OPENSHIFT_DYNAMIC_ENABLED_FLAG, OPENSHIFT_DYNAMIC_ENABLED_DEFAULT);
+
+		if ( dynamicEnabled ) {
```

Comment:
That is more simple, I can do that... i was wanting it to explicitly say 'enabled' or 'disabled'